src/hugchat/cli.py

```diff
@@ -75,7 +79,7 @@ def handle_command(chatbot: ChatBot, userInput: str) -> None:
 
             for i, id_string in conversation_dict.items():
                 info = chatbot.get_conversation_info(id_string)
-                print(f"{i}: ID: {info.id}\nTitle: {info.title[:43]}...\nModel: {info.model}.\nSystem Prompt: {info.system_prompt}\n--------------------------------------------------------")
+                print(f"{i}: ID: {info.id}\nTitle: {info.title[:43]}...\nModel: {info.model}.\nSystem Prompt: {info.system_prompt.split(',')[0]}\n--------------------------------------------------------")
```

Comment:
This might interfere if the user has set their own system prompt instead of using a preset